### DISCUSSION TITLE
Which component library does Material for MkDocs use?

### DISCUSSION THREAD

**Discussion by hhoeflin** [squidfunk/mkdocs-material#4331] [Q&A] [answered]
Hi,

I am not very experienced in web development so please excuse if this question doesn't make much sense. 

I was wondering which version of material is packaged and if all components are usable or if i need to include them separately if i want to use them in embedded html?

Thanks

**Comment by squidfunk** [comment, marked as answer]
I hope I understood your question correctly. Material for MkDocs doesn't use Material UI or another component library – it's all written from scratch. If you install Material for MkDocs, you can use all components that are explained in our documentation, except those that are marked with "Sponsors only". If you want to use those as well, [you need to become a sponsor](https://squidfunk.github.io/mkdocs-material/insiders/#how-to-become-a-sponsor).

**Reply by hhoeflin** [reply]
Yes you understood correctly and thank you for your answer!
Just out of curiosity, what is the reason für not using material ui?

**Reply by squidfunk** [reply]
It's bloated, and most of the components we have need to be custom builds anyway. Also, you can't really use Material components form Markdown, we need to fit the JavaScript to the HTML we get from the Markdown extensions.

**Comment by hhoeflin** [comment]
Makes sense. Thanks! Love your project!
